src/Uno.UI/UI/Xaml/BorderHelper.skia.cs

```diff
@@ -47,13 +47,23 @@ public static void UpdateAllBorderProperties(this IBorderInfoProvider @this)
 		@this.UpdateBorderThickness();
 	}
 
-	public static void SetUpBrushTransitionIfAllowed(BorderVisual visual, Brush fromBrush, Brush toBrush, BrushTransition transition)
+	public static void SetUpBrushTransitionIfAllowed(BorderVisual visual, Brush fromBrush, Brush toBrush, BrushTransition transition, bool isAnimation)
 	{
 		if (transition is null)
 		{
 			return;
 		}
 
+		if (isAnimation)
```

Comment:
As this method is ported code, best to mark with Uno-specific comment. Still unclear to me why we need Uno-specific code though. What exactly is missing to match WinUI?

Reply:
What the comment says:
```
// When an animation sets the background, it skips the transition logic and is applied immediately.
// However, it "deactivates" the currently active transition if one exists. Once the animation is done,
// the deactivated transition is reactivated and continues as if it was running (i.e. it's not paused,
// but takes into account the duration during which it was deactivated).
```